crates/proto/Cargo.toml

```diff
@@ -25,45 +25,61 @@ categories.workspace = true
 license.workspace = true
 
 [features]
-std = ["critical-section/std", "rand/thread_rng"]
-
-tls-aws-lc-rs = ["tokio-rustls/aws-lc-rs", "__tls"]
-https-aws-lc-rs = ["tls-aws-lc-rs", "__https"]
+std = [
+    "critical-section/std",
+    "data-encoding/std",
+    "futures-channel/std",
+    "futures-io/std",
+    "futures-util/std",
+    "ipnet/std",
+    "rand/std",
+    "rand/thread_rng",
+    "ring/std",
+    "thiserror/std",
+    "tracing-subscriber/env-filter",
+    "tracing-subscriber/fmt",
+    "tracing-subscriber/std",
+    "tracing/std",
+    "url/std",
+]
+
+tls-aws-lc-rs = ["std", "tokio-rustls/aws-lc-rs", "__tls"]
```

Comment:
Okay, sorry, one more nit: I think I explained this in an earlier PR, but all of the `tls-*`, `https-*`, `quic-*`, `h3-*` depend on other features that here also include `std -- so there end up being a bunch of double (or triple) transitive enablements of `std`. Can you thin that out a bit to remove the obvious cases at least?

Reply:
Done